tests/test_auth.py

```diff
@@ -216,7 +219,7 @@ def test_update_update_missing(client):
             'name': "Created",
         }
     )
-    assert_response(response, 204)
```

Comment:
it's been a while since I looked at this PR, so I might not be able to argument as well. I wonder, tho, why this test expected 204 here. that seems wrong